workshops/templates/workshops/all_profileupdaterequests.html

```diff
@@ -17,7 +17,7 @@
     <tbody>
     {% for req in requests %}
       <tr>
-        <td>{{ req.personal }} {{ req.family }}</td>
+        <td>{{ req.personal }} {{ req.middle }} {{ req.family }}</td>
```

Comment:
Since you repeatedly need `{{ req.personal}} {{req.middle}} {{req.family}}`, you can create a `person_name` property function in the model and use it instead.